utils.sh

```diff
@@ -63,19 +67,18 @@ get_repository_name () {
 }
 
 get_config_field () {
-  grep "${2}=" < "${package_base}/${1}/.config" | cut -d= -f2
+  grep "${2}=" < "${package_base}/packages/${1}/.${package_name}" | cut -d= -f2
 }
 
 clone_cli_repository () {
-  ( mkdir -p "${package_cloned}" &&
-    cd "${package_cloned}" &&
+  ( cd "${package_clones_path}" &&
     git clone "${1}" ) || exit
 }
 
 go_to_cloned_repo () {
-  cd "${package_cloned}/${1}" || exit
+  cd "${package_clones_path}/${1}" || exit
 }
 
-clear_cloned () {
-  rm -rf "${package_cloned:?}"/*
+clear_clones () {
+  rm -rf "${package_clones_path:?}"/*
```

Comment:
shouldn't be an issue in modern GNU system, but you never know, I would double check that package_clones_path is nonempty before proceeding

Reply:
good idea

Reply:
it is exported in main though, might still be a problem?